webpack/assets/javascripts/react_app/components/AutoComplete/__tests__/integration.test.js

```diff
@@ -5,7 +5,15 @@ import { AutoCompleteProps } from '../AutoComplete.fixtures';
 import Autocomplete, { reducers } from '../index';
 
 jest.mock('../../../API');
-jest.mock('lodash/debounce', () => jest.fn(fn => fn));
+jest.mock('lodash', () => ({
```

Comment:
why can't we mock lodash? this seems like a lot of work for mocking? is this going to be common?

Reply:
We can mock lodash globally.
IMO we should rethink about how we do mocking.

Anyway, I would prefer it not to be apart of this PR if it's fine.